src/platform/pantherlake/include/platform/lib/shim.h

```diff
@@ -0,0 +1,58 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022-2024 Intel Corporation.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *         Keyon Jie <yang.jie@linux.intel.com>
+ *         Rander Wang <rander.wang@intel.com>
+ */
+
+#ifdef __SOF_LIB_SHIM_H__
+
+#ifndef __PLATFORM_LIB_SHIM_H__
+#define __PLATFORM_LIB_SHIM_H__
+
+#include <rtos/bit.h>
+#include <sof/lib/memory.h>
+
+/** \brief Request HP RING Oscillator Clock */
+#define SHIM_CLKCTL_RHROSCC		BIT(31)
```

Comment:
I whined about this earlier, but just to repeat with a little more clarity: AFAICT this whole file is dead code.  These macros aren't expanded anywhere in the existing SOF tree nor the new code in thie PR, and the drivers responsible for this stuff live in Zephyr and have their own API definitions.

Reply:
Agree, dead code and actually we still have such files for previous ACE platforms so cleanup is definitely needed here